package.json

```diff
@@ -2,6 +2,7 @@
   "name": "pancake-frontend",
   "version": "0.1.0",
   "private": true,
+  "type": "commonjs",
```

Comment:
Any reason on this type?

Reply:
no lol had this in because i made a test file ages ago when testing the next apis. they were js files so i added this to get around the require compike issue. always panned to remove